solc/CommandLineInterface.cpp

```diff
@@ -187,27 +186,27 @@ void CommandLineInterface::handleBinary(string const& _contract)
 			createFile(m_compiler->filesystemFriendlyName(_contract) + ".bin-runtime", objectWithLinkRefsHex(m_compiler->runtimeObject(_contract)));
 		else
 		{
-			sout() << "Binary of the runtime part:" << endl;
-			sout() << objectWithLinkRefsHex(m_compiler->runtimeObject(_contract)) << endl;
+			sout() << "Binary of the runtime part:\n";
+			sout() << objectWithLinkRefsHex(m_compiler->runtimeObject(_contract)) << "\n";
 		}
 	}
 }
 
-void CommandLineInterface::handleOpcode(string const& _contract)
+void CommandLineInterface::handleOpcode(std::string const& _contract)
 {
 	solAssert(CompilerInputModes.count(m_options.input.mode) == 1);
 
 	if (!m_options.output.dir.empty())
 		createFile(m_compiler->filesystemFriendlyName(_contract) + ".opcode", evmasm::disassemble(m_compiler->object(_contract).bytecode, m_options.output.evmVersion));
 	else
 	{
-		sout() << "Opcodes:" << endl;
-		sout() << uppercase << evmasm::disassemble(m_compiler->object(_contract).bytecode, m_options.output.evmVersion);
-		sout() << endl;
+		sout() << "Opcodes:\n";
```

Comment:
No changes other than the addition of `std::` please - it makes it significantly more difficult to review, especially in larger PRs.

Reply:
Well, I'm uncertain if this really makes the diff significantly harder to review since all the places I changed are pretty similar and GitHub already highlights what exactly has been added/deleted. I prefer to keep them as is but I can also revert them back if you insist ;)

Reply:
This one, not so much, since it's only three files; the other ones you opened would be significantly more difficult. The reason being is that you're mostly gonna be using automatic substitution, e.g. `string` -> `std::string`, which then may change actual string values e.g. `"string parameter not allowed"`, or comment descriptions, which is something we definitely do not want. So yes, please revert the new line changes; I don't however see anything necessarily wrong with replacing `endl` with `\n`, in fact, it should be preferable *not* to flush the stream in most cases, so if you want, you can open a separate PR for this change (@cameel any objections?).

Reply:
OK, I reverted these changes, can you check again?